### DISCUSSION TITLE
Create public key from 65 bytes of ANSI X9.62 key material

### DISCUSSION THREAD

**Discussion by pke** [paulmillr/noble-curves#126] [Q&A] [answered]
Is it possible to create a public key from 65 bytes of ANSI X9.62 key material?

`0x046b6d7cffa58cb993407f071ef9e7a7b9785b4f6788e0919ebe852b8c209355a520d2fd48bd78319df0de05c5bada36ea89382ace51d6b0b2723cc201d228014e`

**Comment by paulmillr** [comment, marked as answer]
use Point.fromHex() but strip “0x”
